device-detector/tests/test_main.py

```
@@ -0,0 +1,293 @@
+"""Tests for Device Detector Main Service"""
+import pytest
```

Flake8 (7.2.0)

[error] 'pytest' imported but unused

(F401)